head/firmware/main.cpp

```diff
@@ -249,6 +250,21 @@ auto timer_for_notifier = freertos_timer::FreeRTOSTimer(
     }),
     100);
 
+// Unfortunately, these numbers need to be literals or defines
+// to get the compile-time checks to work so we can't actually
+// correctly rely on the hal to get these numbers - they need
+// to be checked against current configuration. However, they are
```

Comment:
Should this comment be in every file that creates a bit timing?

Reply:
I mean maybe but also it's so much